examples/check_integration_error.py

```diff
@@ -53,29 +57,17 @@
     data = clean_data.C
     data.add_noise(fake_data_noise_std)
     # at this point, the fake data has been generated
-    data.ft(["ph1", "ph2"])
-    # {{{ usually, we don't use a unitary FT -- this makes it unitary
-    data /= 0.5 * 0.25  # the dt in the integral for both dims
-    data /= sqrt(ndshape(data)["ph1"] * ndshape(data)["ph2"])  # normalization
-    # }}}
+    data.ft(["ph1", "ph2"],unitary=True)
+    # {{{ vector normalize
     dt = diff(data.getaxis("t2")[r_[0, 1]]).item()
     data.ft("t2", shift=True)
-    # {{{
+    # }}}
     data /= sqrt(ndshape(data)["t2"]) * dt
```

Comment:
this is the line I include in integration_with_error.py as well but unsure if this is problematic/contradicts what we are saying

Reply:
I assume this is also obsolete with my recent push?

Reply:
I assume this is also obsolete with my recent push?